labeled_data/README.md

```diff
@@ -0,0 +1,29 @@
+In this folder, we collect different categories of labeled data. We can using these data by adding `labelIntersect` parameter in metrics like:
+``` 
+openDigger.index.activity.getRepoActivity({labelIntersect: ['Company', ':regions/China'], startYear, startMonth, endYear, endMonth, groupBy: 'Company', groupTimeRange: 'year', limit: -1, order: 'DESC' })
+```
+We have 6 categories of labeled data.
+
+#### Data Description
```

Comment:
Why use a 4th level title when there are no other levels?

Reply:
I'll fix this.